### PR TITLE
Add github templates (.github files)

### PR DESCRIPTION
**Pull request by corycaywood**
Added Issue templates (bug and feature), contribution guide w/ commit message guidelines, and pull request template.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+### Type
+Must be one of the following:
+
+* **build**: Changes that affect the build system or external dependencies (example scopes: gradle, fastlane, npm)
+* **ci**: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+* **docs**: Documentation only changes
+* **feat**: A new feature
+* **fix**: A bug fix
+* **perf**: A code change that improves performance
+* **refactor**: A code change that neither fixes a bug nor adds a feature
```
Comment (ts-nitish-a-jain) [inline .github/CONTRIBUTING.md]: maybe `refactor` should be the last one in types, with a description, anything other than above 😆

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (ts-nitish-a-jain) [inline .github/ISSUE_TEMPLATE/bug_report.md]: your -> the ?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# Contributing
+## Found a Bug?
+If you find a bug in the source code, you can help us by submitting an issue to our GitHub Repository. Even better, you can submit a Pull Request with a fix.
```
Comment (ts-nitish-a-jain) [inline .github/CONTRIBUTING.md]: Do we make the same? 

On the iOS side, there is even a tutorial link:
`**Considering making your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)`

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (corycaywood) [reply .github/CONTRIBUTING.md]: I have updated this.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (corycaywood) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I have updated this.

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (corycaywood) [reply .github/CONTRIBUTING.md]: Sure, we can add this. I just copied these from Remote Config, so there are minor differences between the two.

**Pull request merged**